app/assets/javascripts/application.js

```diff
@@ -88,7 +88,7 @@ if (!String.prototype.supplant) {
 
     moj.Modules.Autocomplete.typeaheadKickoff(insertedSelect);
     moj.Modules.Autocomplete.typeaheadBindEvents(typeaheadWrapper);
-    moj.Modules.FeeFieldsDisplay.addFeeChangeEvent($insertedItem.find('.fx-fee-group'))
+    moj.Modules.FeeFieldsDisplay.addFeeChangeEvent(insertedItem)
```

Comment:
unsure if this should be `$insertedItem` or is using the arg `insertedItem` passed directly ok?

Reply:
Yes - this is correct